src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -47,23 +51,31 @@ public CommandResult(String feedbackToUser, Eatery eateryToShow,
      * and other fields set to their default value.
      */
     public CommandResult(String feedbackToUser) {
-        this(feedbackToUser, null, false, false, false);
+        this(feedbackToUser, null, null,
+                false, false, false);
```

Comment:
why is this split into two lines ah i don't think it'll exceed the max character limit for a line right HAHA